PROJECT_CHARTER_TEMPLATE.md

```diff
@@ -0,0 +1,101 @@
+# ${PROJECT} Charter
+
+_note: the purpose of a project charter is to provide a brief introduction_
+_to the project from a technical, community, or business perspective. The_ 
+_document also connects a project's community leadership and governance with the_
+_OpenJS Foundation's governance._
+
+
+## Section 0: Guiding Principles (optional)
```

Comment:
How do we want projects to handle these optional sections? IMO, the sections should still exist in every charter and if the project chooses not to complete it, they leave it blank or add N/A or something like that so that all charters have consistent numbering, sections, order, etc.

Reply:
I've seen the phrase 'Intentionally Omitted' or 'Section Intentionally Left Blank' used for that purpose, if that works

Reply:
👍 Works for me as long as the section heading remains to maintain consistency.

Reply:
Of the 2 suggestions I prefer  'Section Intentionally Left Blank'

Reply:
I pushed a commit that adds the sentence: "If you are not using this section, please indicate your intent with the phrase, 'Section Intentionally Left Blank'." to the optional sections